cli/dts/lib.deno.ns.d.ts

```diff
@@ -94,6 +94,16 @@ declare namespace Deno {
    * See: https://no-color.org/ */
   export const noColor: boolean;
 
+  export interface TestPermissions {
+    read?: string[] | boolean;
```

Comment:
Not too sure about this overload, primarily added it because unitTest has the boolean variant and I want to be able to specify specific directories in addition to just a blanket "allow write".

Reply:
Sounds good to me!

Reply:
Actually, should consider if there's something to be shared here between these permissions and worker permissions.